care/facility/migrations/0076_auto_20200407_0245.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.11 on 2020-04-06 21:15
+
+from django.db import migrations
+import multiselectfield.db.fields
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'multiselectfield.db.fields'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5316739)

Reply:
This should be related to the set up of the environment.